pyproject.toml

```diff
@@ -76,3 +76,8 @@ skip-string-normalization = '1'
         directory = "change"
         name = "Changes"
         showcontent = true
+
+[[source]]
```

Comment:
I released the new version of jsondaora on pypi: https://pypi.org/project/jsondaora/0.8.0/